ERROR: support/rrsync.1 cannot be created.

when md2man cannot create manpages (configure check tells "checking if md2man can create manpages... no") and rsync is build with "--with-rrsync" configure switch ,  make bails out with

```
gcc -I. -I. -I./zlib -g -O2 -DHAVE_CONFIG_H -Wall -W  -c zlib/compress.c -o zlib/compress.o
gcc -I. -I. -I./zlib -g -O2 -DHAVE_CONFIG_H -Wall -W  -c zlib/crc32.c -o zlib/crc32.o
gcc -I./zlib -g -O2 -DHAVE_CONFIG_H -Wall -W  -o rsync flist.o rsync.o generator.o receiver.o cleanup.o sender.o exclude.o util1.o util2.o main.o checksum.o match.o syscall.o log.o backup.o delete.o options.o io.o compat.o hlink.o token.o uidlist.o socket.o hashtable.o usage.o fileio.o batch.o clientname.o chmod.o acls.o xattrs.o progress.o pipe.o  simd-checksum-x86_64.o  params.o loadparm.o clientserver.o access.o connection.o authenticate.o lib/wildmatch.o lib/compat.o lib/snprintf.o lib/mdfour.o lib/md5.o lib/permstring.o lib/pool_alloc.o lib/sysacls.o lib/sysxattrs.o  zlib/deflate.o zlib/inffast.o zlib/inflate.o zlib/inftrees.o zlib/trees.o zlib/zutil.o zlib/adler32.o zlib/compress.o zlib/crc32.o  -lattr -lacl -lpopt -llz4 -lzstd -lxxhash -lcrypto
cp -p ./support/rrsync rrsync
ERROR: rsync.1 cannot be created.
make: *** [Makefile:257: rsync.1] Error 1
```

i can reproduce this on linux, whereas  i came across this when adding "--with-rrsync" to rsync.rb in homebrew project. automated build/ci failed after that.  it seems macos einvironment does not have proper python dependencies for that , though brew formulae are installed. 

```
% ./md-convert --test rsync-ssl.1.md
Failed to find cmarkgfm or commonmark for python3.
```

```
% brew search cmark
==> Formulae
cmark ✔                                      cmark-gfm ✔
```

```
# find . -name "*.1.md"
./rsync.1.md
./support/rrsync.1.md
./rsync-ssl.1.md
```
```
# find . -name "*.1"
./rrsync.1
./rsync-ssl.1
./rsync.1
```

there is a patch for quite a while in the debian package to adress this, but it looks weird/wrong to me

https://sources.debian.org/patches/rsync/3.4.1+ds1-8~exp1/fix_rrsync_man_generation.patch/

--- a/maybe-make-man
+++ b/maybe-make-man
@@ -15,7 +15,7 @@ if [ ! -f "$flagfile" ]; then
     if "$srcdir/md-convert" --test "$srcdir/rsync-ssl.1.md" >/dev/null 2>&1; then
 	touch $flagfile
     else
-	outname=`echo "$inname" | sed 's/\.md$//'`
+	outname=`basename "$inname" .md`
 	if [ -f "$outname" ]; then
 	    exit 0
 	elif [ -f "$srcdir/$outname" ]; then

